core/services/relay/evm/mercury/wsrpc/fixtures/domain.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
These are fixtures that are for testing only and are fine to be made public

Reply:
Same comment than for `core/services/ocr2/plugins/mercury/fixtures/domain.key`, could we change the name to reflect it is intended for tests?